### ISSUE TITLE
cannot add a ordered/unordered list to a table cell

### ISSUE THREAD

**Issue by artknight** [soccerloway/quill-better-table#104]
When trying to add a list into a table cell, the table breaks into two and the list is added as a outside component

<img width="325" alt="Screenshot 2024-07-17 at 11 54 31 AM" src="https://github.com/user-attachments/assets/7e7d2198-23aa-44d1-b9c1-9bb64b7a6269">

**Comment by Libpzzz** [comment]
Hello, have you solved this problem？

**Comment by artknight** [comment]
No, I have not. I was hoping the creator @soccerloway would be able to provide an update.
